Podfile.lock

```diff
@@ -75,6 +75,6 @@ SPEC CHECKSUMS:
   stellar-ios-mac-sdk: 7a49cbb604d6ec8e0e3fefddf34d0d9340db2f1c
   Whisper: 8c499b08f3b56d5c80162d775e3462318ad7f4fe
 
-PODFILE CHECKSUM: c16c3a1f64d010dfff91b5763b95469e7be09cbd
+PODFILE CHECKSUM: a20b2a55a46db8619698bfc60bab052b3593e01e
 
-COCOAPODS: 1.6.0.beta.2
+COCOAPODS: 1.5.3
```

Comment:
Yeah here we go... It looks like all the pod file and project file changes are coming from the fact you're using cocoapds 1.5.3, and I've bumped us up to 1.6.0. Getting up to the beta release version and re-running `pod install` should resolve these issues.